CONTRIBUTING.md

```diff
@@ -7,32 +7,41 @@ Thank you for considering contributing to Thyra! We welcome contributions from t
 To contribute to Thyra, you should have a basic understanding of the Go programming language and Git version control system. If you're new to Go, we recommend checking out [A Tour of Go](https://tour.golang.org/welcome/1) to get started.
 
 Before you can start contributing, you'll need to complete the following steps:
+
```

Comment:
Are you using a specific convention here? Can you share it ?

Reply:
yes I use this VSCode extension: https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint